config/http.conf

```diff
@@ -2,12 +2,19 @@ http {
   include /etc/nginx/mime.types;
   include /etc/nginx/log.conf;
 
-  server_tokens         off;
-  keepalive_timeout     20s;
-  sendfile              on;
-  tcp_nopush            on;
-  client_max_body_size  400m;
-  client_body_timeout   300s;
+  server_tokens           off;
+  sendfile                on;
+  tcp_nopush              on;
+
+  client_max_body_size    500m;
+  client_body_buffer_size 128k;
```

Comment:
What's the intention behind increasing this?

Reply:
There are some links in the commit for more context but the TL;DR is that it's to:

1. Reduce the number of "a client request body is buffered to a temporary file" warning we get.
2. Reduce the number of times nginx has to write file (search for above string in logs to see). Memory is faster. Also since we have `client_max_body_size` also figured uploads would benefit too by having to write even less.

It's not that great of an issue, more like a nice-to-have rather than must-have. FYI, you can see evidence of this below ...

<img width="1389" alt="Screenshot 2021-02-25 at 14 31 40" src="https://user-images.githubusercontent.com/703786/109147590-3a20b380-7776-11eb-9874-36b6f088c3b4.png">